syntax = "proto3";

package youtubesummary;

message YoutubeSummaryRequest {
  string user_id = 1;
  string youtube_context = 2;
}

message YoutubeSummaryResponse {
  string content = 1;
  string is_final = 2;
}

service YoutubeSummaryService {
  rpc YoutubeSummary (YoutubeSummaryRequest) returns (stream YoutubeSummaryResponse);
}